core/sql/executor/ExExeUtilCli.cpp

```diff
@@ -1119,7 +1119,7 @@ Lng32 ExeCliInterface::executeImmediate(const char * stmtStr,
     {
       // Allocate the diagnostics area if needed
       // and populate the diagnostics conditions
-      if (*globalDiags == NULL && retcode != 0) {
+      if (*globalDiags == NULL && retcode != 0 && retcode != 100) {
```

Comment:
I take it in the old code we would throw away the 100 warning somewhere else?

Reply:
Returning retcode with a value of 100 is ok and should be treated as end of result by the callers. This additional check just avoids creating a ComDiagsArea if the return code is just 100.